src/cprover/find_variables.cpp

```diff
@@ -0,0 +1,40 @@
+/*******************************************************************\
+
+Module: Find Variables
+
+Author: Daniel Kroening, dkr@amazon.com
+
+\*******************************************************************/
+
+/// \file
+/// Find Variables
+
+#include "find_variables.h"
+
+#include <util/pointer_expr.h>
+
+#include "state.h"
```

Comment:
Does this code actually use anything from that include, and should the entire code not be part of `find_symbols.{h,cpp}` instead?

Reply:
I am adding a comment -- note that this is materially different to any variant of find_symbols.

Reply:
I understand, yet this is a concern of mine for several pieces of code being added here: they are different from existing functionality, but the absence of documentation makes it impossible for a potential user to tell what exactly the difference is without putting the code side-by-side.